.github/workflows/userbenchmark-regression-detector.yml

```diff
@@ -24,33 +24,19 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: benchmark
-      - name: Create conda environment
-        run: |
-          . "${SETUP_SCRIPT}"
-          python benchmark/utils/python_utils.py --create-conda-env "${CONDA_ENV_NAME}"
-          sudo python benchmark/utils/cuda_utils.py --setup-cuda-softlink
       - name: Tune Nvidia GPU
         run: |
           . "${SETUP_SCRIPT}"
           sudo nvidia-smi -pm 1
           sudo nvidia-smi -ac 1215,1410
           nvidia-smi
-      - name: Install PyTorch nightly
+      - name: Check PyTorch nightly if scheduled
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
This line means, if the workflow is scheduled by GitHub, it will check the torch nightly package is the latest, and will fail if it is not (currently it will fail because torchvision build failure).

If the workflow is triggered by user (like clicking on the run workflow button), it will skip the check, and use whatever the latest docker version available.

Reply:
Yea...I had blindly copied the code forgetting that this was not the docker pull step. Will be adding that in.

Reply:
See: https://github.com/pytorch/test-infra/blob/xz9/add-gcp-autoscaler/gcp/k8s/runnerdeployment.yaml#L10